tests/integration/220.command.graph.test.js

```diff
@@ -1,8 +1,63 @@
 const test = require('ava')
 
 const callCli = require('./utils/call-cli')
+const { getCLIOptions, withMockApi } = require('./utils/mock-api')
+const { withSiteBuilder } = require('./utils/site-builder')
 const { normalize } = require('./utils/snapshots')
 
+const mongoSite = {
+  account_slug: 'test-account',
+  build_settings: {
+    env: {
+      NETLIFY_GRAPH_PERSIST_QUERY_TOKEN: 'zxcv0987',
+    },
+  },
+  id: 'site_id',
+  name: 'site-name',
+}
+const envelopeSite = {
+  ...mongoSite,
+  use_envelope: true,
+}
+const envelopeResponse = [
+  {
+    key: 'NETLIFY_GRAPH_WEBHOOK_SECRET',
+    scopes: ['functions'],
+    values: [{ context: 'all', value: 'abcd1234' }],
+  },
+  {
+    key: 'NETLIFY_GRAPH_PERSIST_QUERY_TOKEN',
+    scopes: ['functions'],
+    values: [{ context: 'all', value: 'zxcv0987' }],
+  },
+]
+const routes = (site) => [
+  { path: 'sites/site_id', response: site },
+  {
+    path: 'sites/site_id',
+    method: 'PATCH',
+    response: {},
+  },
+  {
+    path: 'accounts',
+    response: [{ slug: site.account_slug }],
+  },
+  {
+    path: 'sites/site_id/service-instances',
+    response: 'uuid-string',
+  },
+  {
+    path: 'accounts/test-account/env',
+    method: 'GET',
+    response: [envelopeResponse[1]],
```

Comment:
In the test, we set `NETLIFY_GRAPH_PERSIST_QUERY_TOKEN` to already exist, in order to skip over some of the logic that reaches out using the OneGraph client. I didn't know how to mock that out, so skipped over it to at least test some of the function.